.vib/minio-client/goss/minio-client.yaml

```diff
@@ -0,0 +1,4 @@
+command:
+  check-app-binary:
+    exec: mc --help
+    exit-status: 0
```

Comment:
Why not checking the version as we usually do? This way, the CLI is also checked and we can remove this test 🚀